src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/service/AdjustmentsTransactionalService.kt

```diff
@@ -414,6 +427,15 @@ class AdjustmentsTransactionalService(
     return null
   }
 
+  fun getSentenceInfo(chargeIds: List<Long>, sentences: List<SentenceAndOffences>): SentenceInfo? {
```

Comment:
Can you name this function a bit more sprecifically. Get sentenceInfoFromChargeIds or something like that.